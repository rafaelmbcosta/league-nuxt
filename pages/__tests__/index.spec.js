import { shallowMount, createLocalVue } from "@vue/test-utils"
import Vuetify from "vuetify"
import index from "../index"

const localVue = createLocalVue()
describe("index", () => {
  test("loads component", () => {
    const vuetify = new Vuetify()
    const wrapper = shallowMount(index, {}, localVue, vuetify)
    expect(wrapper).toMatchSnapshot()
  })
})
