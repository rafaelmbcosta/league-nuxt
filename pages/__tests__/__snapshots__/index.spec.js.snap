// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`index loads component 1`] = `
<div>
  <v-row-stub tag="div">
    <v-col-stub cols="12" tag="div">
      <topscores-stub></topscores-stub>
    </v-col-stub>
  </v-row-stub>
  <v-row-stub tag="div">
    <v-col-stub cols="12" tag="div">
      <scores-stub></scores-stub>
    </v-col-stub>
  </v-row-stub>
</div>
`;
